pkg/disk/btrfs.go

```diff
@@ -86,8 +85,7 @@ type BtrfsSubvolume struct {
 	Size       uint64
 	Mountpoint string
 	GroupID    uint64
-
-	MntOps string
+	Compress   string
```

Comment:
I'm not sure about this change.  I see mount options / fstab options a bit like kernel args.  There's too many to cover as individual options.  We're good with just compression for now because that's all that's set by default in Fedora (right?), but are we going to keep adding properties for every option we want to cover in the future?  I suppose we could.

If we do, do we make the same change for the filesystem type?
https://github.com/osbuild/images/blob/8f50ec4b03aec02ea8d7e90ebd1633f4eee47407/pkg/disk/filesystem.go#L18

Reply:
Working on some karg fiddling now and I'm changing my mind.  Let's handle all the args we know about :)